sdk/README.md

```diff
@@ -55,8 +55,12 @@ from pprint import pprint
 
 ## Documentation for API Endpoints
 
-Class | Method | HTTP request | Description
------------- | ------------- | ------------- | -------------
+Class | Method |  Description
+------------ | ------------- | -------------
+[KFServingApi](docs/KFServingApi.md) | [deploy](docs/KFServingApi.md#deploy) | Create the provided KFService in the specified namespace|
```

Comment:
Nit: consider Api -> API.

Reply:
As your below comments, changed to "KFServingClient", thanks.